<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-04T11:43:39 IST" name="Surefire suite" finished-at="2023-06-04T11:43:53 IST" duration-ms="14060">
    <groups>
    </groups>
    <test started-at="2023-06-04T11:43:39 IST" name="Surefire test" finished-at="2023-06-04T11:43:53 IST" duration-ms="14060">
      <class name="tests.LoginTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.LoginTest@75329a49]" started-at="2023-06-04T11:43:39 IST" name="setUp" finished-at="2023-06-04T11:43:43 IST" duration-ms="4142" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="login()[pri:0, instance:tests.LoginTest@75329a49]" started-at="2023-06-04T11:43:43 IST" name="login" finished-at="2023-06-04T11:43:45 IST" duration-ms="1248" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="signOut()[pri:0, instance:tests.LoginTest@75329a49]" started-at="2023-06-04T11:43:45 IST" name="signOut" finished-at="2023-06-04T11:43:46 IST" duration-ms="1043" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signOut -->
        <test-method is-config="true" signature="tear()[pri:0, instance:tests.LoginTest@75329a49]" started-at="2023-06-04T11:43:46 IST" name="tear" finished-at="2023-06-04T11:43:46 IST" duration-ms="293" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tear -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.ManageRoomTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.ManageRoomTest@49c386c8]" started-at="2023-06-04T11:43:46 IST" name="setUp" finished-at="2023-06-04T11:43:48 IST" duration-ms="2044" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addRooms()[pri:0, instance:tests.ManageRoomTest@49c386c8]" started-at="2023-06-04T11:43:48 IST" name="addRooms" finished-at="2023-06-04T11:43:53 IST" duration-ms="4933" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at tests.ManageRoomTest.addRooms(ManageRoomTest.java:31)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addRooms -->
        <test-method is-config="true" signature="tear()[pri:0, instance:tests.ManageRoomTest@49c386c8]" started-at="2023-06-04T11:43:53 IST" name="tear" finished-at="2023-06-04T11:43:53 IST" duration-ms="303" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tear -->
      </class> <!-- tests.ManageRoomTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
